src/print.ts

```diff
@@ -247,6 +247,8 @@ class Printer {
     let edgeMustaches = this.formatEdgeProps(node.edgeMustaches);
     let comments = this.formatComments(node.comments);
 
+    const closingTag = node.type == "voidTag" ? " />" : ">";
```

Comment:
This was done on purpose. There is no-space when the closing bracket is in a new-line. However, there is additional space when it is inline.

For example:

Inline

```edge
<meta charset="UTF-8" />
```

Multi-line

```edge
<path
    fill-rule="evenodd"
    clip-rule="evenodd"
    d="M10 26c0 12.904 3.096 16 16 16s16-3.096 16-16-3.096-16-16-16-16 3.096-16 16Z"
    fill="#8E52A9"
  />
```